src/Arcus.Messaging.Pumps.Abstractions/IRestartableMessagePump.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Messaging.Pumps.Abstractions
+{
+    /// <summary>
+    /// Represents an <see cref="MessagePump"/> that can be restarted programmatically.
+    /// </summary>
+    public interface IRestartableMessagePump : IHostedService
+    {
+        /// <summary>
+        /// Gets the unique message pump ID to identify the pump that needs to be restarted.
+        /// </summary>
+        string JobId { get; }
```

Comment:
Again, the property is called `JobId`, but it is a member of a MessagePump class, and in the comments it is also refered to as 'message pump id'.

Reply:
We already had a `JobId` as a property on the pump that was identifying it as such. The reason we choose 'job' is probably to identify any type of hosted service. Also, some kind of background job. We also have already a `Id` property on the message pump (not sure why, honestly), so if we would add another one, it would maybe make this more complicated, no?